docs/self-managed/concepts/secondary-storage/index.md

```diff
@@ -0,0 +1,71 @@
+---
+id: index
+title: "Secondary storage"
+description: "Learn how secondary storage works in Camunda Self-Managed environments, how it interacts with the Zeebe engine, and how to configure or manage it effectively."
+---
+
+Camunda uses a layered storage model that separates workflow execution data from data used by web applications and APIs. This section explains how secondary storage works, how it connects to the Zeebe engine, and how to configure or maintain it in Self-Managed environments.
+
+## How secondary storage fits into Camunda
+
+Camunda’s data model consists of two complementary layers:
+
+| Layer                 | Purpose                                                                                        | Typical technologies        | Used by                               |
+| --------------------- | ---------------------------------------------------------------------------------------------- | --------------------------- | ------------------------------------- |
+| **Primary storage**   | Persists real-time workflow execution state managed by Zeebe.                                  | RocksDB (embedded in Zeebe) | Zeebe broker                          |
+| **Secondary storage** | Stores exported workflow, decision, and task data for querying, visualization, and API access. | Elasticsearch, H2, or RDBMS | Operate, Tasklist, Identity, REST API |
+
+:::note
+Secondary storage is not a duplicate of primary data. It is a projection optimized for search, analytics, and human-task interaction.
+:::
+
+## Why it matters
+
+- **Performance**: Offloads analytical and API queries from the Zeebe engine.
+- **Flexibility**: Supports multiple backend technologies, such as Elasticsearch or relational databases.
+- **Resilience**: Can be backed up and scaled independently from the workflow engine.
+
+## Supported storage options
+
+Camunda supports several secondary storage backends, depending on the version and use case:
+
+| Database type                         | Availability         | Use case                                               |
+| ------------------------------------- | -------------------- | ------------------------------------------------------ |
+| **Elasticsearch**                     | General availability | Default in most production installations.              |
+| **H2**                                | 8.9-alpha1+          | Lightweight option for local testing or Camunda 8 Run. |
+| **RDBMS** (PostgreSQL, MariaDB, etc.) | In development       | For future enterprise-grade flexibility.               |
```

Comment:
Ok now I understand the distinction as it relates to use case. I still don't think it's correct to separate H2 from RDBMS though as though they are different things entirely, because it will confuse users.

Reply:
Applied the same fix in the supported storage table too 👍